lib/room.js

```diff
@@ -263,6 +265,15 @@ class Room extends EventEmitter {
  * @event Room#trackMessage
  */
 
+/**
+ * A {@link RemoteTrack} was published by a {@link RemoteParticipant} to the {@link Room}.
```

Comment:
We may want to call out that this is _not_ raised for LocalParticipant events.

Reply:
@markandrus Isn't it obvious from the presence of `RemoteParticipant` and `RemoteTrack` in the description?